README.md

```diff
@@ -10,15 +10,15 @@
 | BROKER_CLIENT_DIRECT_ENABLED | Enables the direct client. Possible values are `true` and `false`.                                                                                                     |                  | `false`                                                              |
 | BROKER_CLIENT_AKTIN_ENABLED  | Enables the aktin client. Possible values are `true` and `false`.                                                                                                      |                  | `false`                                                              |
 | BROKER_CLIENT_DSF_ENABLED    | Enables the dsf client. Possible values are `true` and `false`.                                                                                                        |                  | `false`                                                              |
-| KEYCLOAK_ENABLED             | Enables Keycloak if set to true. Possible values are `true` and `false`.                                                                                               |                  | `true`                                                               |
-| KEYCLOAK_BASE_URL            | Base URL to reach a keycloak instance.                                                                                                                                 |                  | `http://localhost:8080`                                              |
+| KEYCLOAK_ENABLED             | Enables keycloak if set to true. Possible values are `true` and `false`.                                                                                               |                  | `true`                                                               |
```

Comment:
@michael-82  - keycloak enabling and disabling does not exist anymore => remove from readme